modules/network-observability-RTT.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-RTT_{context}"]
+= Working with RTT tracing
+You can track RTT by editing the `FlowCollector` to the specifications in the following YAML example.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. In the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+. Select *cluster*, and then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource for RTT tracing, for example:
++
+[id="network-observability-flowcollector-configuring-RTT_{context}"]
+.Example `FlowCollector` configuration
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: DIRECT
+  agent:
+    type: EBPF
+    ebpf:
+      features:
+       - FlowRTT               <1>
+----
+<1> You can start tracing RTT network flows by listing the `FlowRTT` parameter in the `spec.agent.ebpf.features` specification list.
+
+.Verification
+* When you refresh the *Network Traffic* page, the *Overview*, *Traffic Flow*, and *Topology* views display new information about RTT:
+.. Select new choices in *Manage panels* to choose which graphical visualizations of RTT to display in the *Overview*.
+.. In *Manage columns*, choose to display the *Flow RTT* column in the *Traffic flows* table.
```

Comment:
`Flow RTT` column is shown by default when the feature is enabled; no need to choose in Manage columns

Reply:
Even if it's shown by default, it's good to know you can manage the display there.

Reply:
yeah then better to reword in such a way saying that `a new FlowRTT column is seen`